packages/klevu-ui/src/components/klevu-product/klevu-product.tsx

```diff
@@ -136,6 +141,12 @@ export class KlevuProduct {
       if (this.showAddToCart === undefined) {
         this.showAddToCart = settings.klevu_addToCartEnabled
       }
+      if (this.hideHoverImage === undefined) {
+        this.hideHoverImage = !Boolean(settings.klevu_uc_userOptions.showRolloverImage)
```

Comment:
the values are boolean, what benefit do we get by converting them to Boolean?

Reply:
Making sure that the value is Boolean. Not undefined.